classes/Domain/Repository/UserRepository.php

```diff
@@ -11,28 +11,28 @@
  * @see https://github.com/opencfp/opencfp
  */
 
-namespace OpenCFP\Domain\Speaker;
+namespace OpenCFP\Domain\Repository;
```

Comment:
I think it could be useful to add more repositories here (and actual implementations), so that we can use those in controllers, instead of using static methods to retrieve models. This would have the advantage that more tests can be turned into unit tests.

Reply:
Yes, correctly grouping them as repositories would be good.